src/OpenLoco/Config.cpp

```diff
@@ -139,6 +139,8 @@ namespace OpenLoco::Config
             _new_config.language = config["language"].as<std::string>();
         if (config["breakdowns_disabled"])
             _new_config.breakdowns_disabled = config["breakdowns_disabled"].as<bool>();
+        if (config["trains_reverse_at_signals"])
+            _new_config.trains_reverse_at_signals = config["trains_reverse_at_signals"].as<bool>();
```

Comment:
New config options should use camelCase. We're in an awkward transition phase, but eventually all options should be using camelCase.

Reply:
Done. If there's interest, I'd be happy to refactor the other option names in another PR?

Reply:
Thank you for offering! The trouble is compatibility with older config files. Ideally, we would find a nice solution that does not involve duplicating the loading routine. It's already quite verbose as-is.

There has been some talk about moving to a JSON based format instead, as the libraries available are much more actively developed. Therefore, perhaps the conversion is best left for when we switch formats. (Feel free to nudge us on Discord about this sometime.)

Reply:
No worries! By older config files, do you mean existing `openloco.yml` files on people's computers or original locomotion files?

Yeah, config loading could do with a bit of optimisation. No promises, but I'd be keen to have a stab at refactoring.

Also tempted to look at adding some more structure to the keyboard shortcuts config entries. I have a feeling that things could get out of hand pretty quickly with the flat structure if we add shortcuts for more things from other areas of the game. I wouldn't go too nuts - just nesting/grouping logically.